### DISCUSSION TITLE
model onnx with spatial=0 not support in ONNX runtime

### DISCUSSION THREAD

**Discussion by luan1412167** [apache/incubator-mxnet#16405] [Q&A] [answered]
## Description
Hi, I'm converting arcface model from mxnet to onnx. However, mx2onnx convert model with spatial=0 when onnx runtime not support to run with model spatial=0.
I have change spatial parameter in code (mxnet/contrib/onnx/mx2onnx/_op_translations.py)
and reshape prelu input and add flatten layer.
Then it can run but the result is wrong. For more detail informations I have opened issue on onnx runtime repo [here](https://github.com/microsoft/onnxruntime/issues/2045#issue-504081725)

What is my wrong when get the wrong result?

(Brief description of the problem in no more than 2 sentences.)

## Environment info (Required)
 mxnet 1.5.0 numpy 1.17.2 onnx 1.6.0 onnxruntime 0.5.0
Package used (Python/R/Scala/Julia):
Python 3.7

**Comment by ddavydenko** [comment]
@mxnet-label-bot, add [Question]

**Comment by szha** [comment, marked as answer]
This is an issue with ONNX runtime and has been resolved in the linked issue.
